specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/GalleryRP.json

```diff
@@ -5485,6 +6242,273 @@
+      "enum": [
+        "String",
+        "Int",
+        "Double",
+        "Boolean",
+        "Enum"
+      ],
+      "x-ms-enum": {
+        "name": "GalleryScriptParameterType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "String",
+            "value": "String",
+            "description": "String gallery script parameter type"
+          },
+          {
+            "name": "Int",
+            "value": "Int",
+            "description": "Int gallery script parameter type"
+          },
+          {
+            "name": "Double",
+            "value": "Double",
+            "description": "Double gallery script parameter type"
+          },
+          {
+            "name": "Boolean",
+            "value": "Boolean",
+            "description": "Boolean gallery script parameter type"
+          },
+          {
+            "name": "Enum",
+            "value": "Enum",
+            "description": "Enum gallery script parameter type"
+          }
+        ]
+      }
+    },
+    "GalleryScriptProperties": {
+      "type": "object",
+      "description": "Describes the properties of a gallery script definition.",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of this gallery script definition resource. This property is updatable."
+        },
+        "eula": {
+          "type": "string",
+          "description": "The Eula agreement (End User License Agreement) for the gallery Script Definition."
+        },
+        "privacyStatementUri": {
+          "type": "string",
+          "description": "The privacy statement uri."
+        },
+        "releaseNoteUri": {
+          "type": "string",
+          "description": "The release note uri."
+        },
+        "endOfLifeDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The end of life date of the gallery Script Definition. This property can be used for decommissioning purposes. This property is updatable."
+        },
+        "supportedOSType": {
+          "$ref": "#/definitions/OperatingSystemTypes",
+          "description": "This property allows you to specify the supported type of the OS that application is built for. Possible values are: **Windows,** **Linux.**"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/GalleryProvisioningState",
+          "description": "The provisioning state, which only appears in the response.",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "supportedOSType"
+      ]
+    },
+    "GalleryScriptUpdate": {
+      "type": "object",
+      "description": "Specifies information about the gallery Script Definition that you want to update.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GalleryScriptProperties",
```

Comment:
@viveklingaiah
I am getting this error:
<img width="1352" height="127" alt="image" src="https://github.com/user-attachments/assets/c5ab75f8-2235-4017-b969-22bc52ef2317" />
https://github.com/Azure/azure-rest-api-specs/actions/runs/16300595655

should we create a model called "GalleryScriptUpdateProperties" that does not have provisioningState?